README.md

```diff
@@ -32,14 +33,27 @@
 # Provable Refactorings
 
 If you can get a series of commits that is all lowercase commits, you can deploy without the need for Regression Testing, or lengthy conversations about accepting the pull request to trunk.
-[Recipes Here](https://github.com/InnovatingTeams/provable-refactorings)
 
-A provable refactoring requires the burden of proof. The main methods of proof are
-* automated refactoring via tool
-* Scripted manual refactoring, using the compiler to verify each step
-* DANGER: Very Highly tested code, with the tests providing proof
+A provable refactoring requires a burden of proof. The main methods of proof are
+* automated refactoring via tool, with knowledge of tool bugs.
+* Scripted manual refactoring, using the compiler to verify each step. [Recipes Here](https://github.com/InnovatingTeams/provable-refactorings)
 
-Note that only the first two levels can prove bug-for-bug copmatability. The last can only demonstrate that you didn't cause any problems that have been thought of before; it does not demonstrate safety for novel bugs.
+With discipline these can prove bug-for-bug compatibility. They demonstrate safety for unknown bugs, even guaranteeing that you do not accidentally fix a bug you don't know exists (but your customers may be depending on).
+
+All of these recipes use static analysis to demonstrate safety. As such, they work equally well on code that lacks tests. They can be a good way to make code testable. Their downside is that they are language-specific.
+
+# Test-supported Refactorings
+[3]:#test-supported-refactorings
+
+These are refactorings with a lower standard of proof:
+1. Commit contains only a single refactoring.
+2. Refactoring is named and published (e.g., in [Fowler's refactoring catalog](https://refactoring.com/catalog/)).
+3. Your entire product is very highly tested.
+4. You followed the published recipe, including running full-suite test runs when indicated.
+
+Note that this can not prove bug-for-bug compatibility. It can only demonstrate that you didn't cause any problems that have been thought of before; it does not demonstrate safety for novel bugs.
```

Comment:
This kind of refactoring is fine (safe enough) in brand new / unpublished / uncalled code. 

(Which is what the TDD literature seems to assume.)